app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from typing import Any
+from .errors import (NotVaccinatedError,
+                     OutdatedVaccineError,
+                     NotWearingMaskError)
```

Comment:
Use absolute imports instead of relative imports as per the checklist. Change to: `from app.errors import NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError`.